### PR TITLE
Modifying Japanese.

### PR DESCRIPTION
**Pull request by PawarenessC**
Hi!

I am Japanese.
I changed some Japanese!

### CODE REVIEW

**Comment by Johnmacrocraft** [comment]
Thanks for your proofreading! I was hoping someone to do it :)
I'll be checking your changes soon (I'm tired right now XD)

File: resources/lang/jpn.ini [hunk 1]
```diff
@@ -18,14 +18,14 @@ gui.buyinfo.button.buy={%0} で購入
...
 gui.buyinfo.error.alreadySubscribe=この新聞を既に購読しています。
-gui.buyinfo.error.noMoney=この新聞を購読するのに十分なお金がありません、{%0} がもっと必要です
-gui.buyinfo.info.invNoSpace=購読に成功しましたが、インベントリに十分なスペースがなくて新聞を追加できません。新聞は、あなたがサーバーに再び参加すると追加されます。
+gui.buyinfo.error.noMoney=この新聞を購読するのに十分なお金がありません、{%0} が必要です
+gui.buyinfo.info.invNoSpace=購読に成功しましたが、インベントリに十分なスペースがなく、新聞を追加できません。新聞は、あなたがサーバーに再び参加すると追加されます。
 gui.buyinfo.success.subscribe={%0} を購読しました！
 
 gui.buyitems.title=新聞を選択
-gui.buyitems.label.noItems=この新聞は、まだ何の新聞も発行していないようです。後で確認してください。
+gui.buyitems.label.noItems=この新聞は、まだ何も発行していないようです。後で確認してください。
```
Comment (Johnmacrocraft) [inline resources/lang/jpn.ini:26]: Does it make more sense to remove the word newspaper from the translation?
Other changes look quite good!

File: resources/lang/jpn.ini [hunk 2]
```diff
@@ -18,14 +18,14 @@ gui.buyinfo.button.buy={%0} で購入
 gui.buyinfo.button.subscribe={%0} で購読
 gui.buyinfo.perm.subscribe=新聞を購読する
 gui.buyinfo.error.alreadySubscribe=この新聞を既に購読しています。
-gui.buyinfo.error.noMoney=この新聞を購読するのに十分なお金がありません、{%0} がもっと必要です
-gui.buyinfo.info.invNoSpace=購読に成功しましたが、インベントリに十分なスペースがなくて新聞を追加できません。新聞は、あなたがサーバーに再び参加すると追加されます。
+gui.buyinfo.error.noMoney=この新聞を購読するのに十分なお金がありません、{%0} が必要です
+gui.buyinfo.info.invNoSpace=購読に成功しましたが、インベントリに十分なスペースがなく、新聞を追加できません。新聞は、あなたがサーバーに再び参加すると追加されます。
```
Comment (Johnmacrocraft) [inline resources/lang/jpn.ini:22]: Not sure if a comma is really needed after なくて.

**Review by Johnmacrocraft — DISMISSED** [review]
Most of the changes look good, except few parts.
(Apparently these reviews were not shown until now because I forgot to submit them... Oof)

File: resources/lang/jpn.ini (hunk 2 above)
Comment (PawarenessC) [reply resources/lang/jpn.ini:22]: need :)

File: resources/lang/jpn.ini (hunk 1 above)
Comment (PawarenessC) [reply resources/lang/jpn.ini:26]: Please delete it.

File: resources/lang/jpn.ini (hunk 2 above)
Comment (Johnmacrocraft) [reply resources/lang/jpn.ini:22]: There are more strings with なくて, you will probably want to add to there too

**Comment by Johnmacrocraft** [comment]
Merged, once again thank you for your translations contribution!

**Pull request merged**